application/templates/deployment.yaml

```diff
@@ -291,4 +297,4 @@ spec:
         {{- else }}
       serviceAccountName: {{ template "application.name" . }}
       {{- end }}
-{{- end }}
\ No newline at end of file
+{{- end }}
```

Comment:
Whitespace changes mess with blame information. Harmless as it is, I'd suggest removing this change any commits that'll be merged.